__tests__/app.test.js

```diff
@@ -102,5 +94,30 @@ describe("app tests", () => {
         })
       })
     })
+  
+    describe("task 4 get comments for specified review", () => {
```

Comment:
ideally we'd also test to ensure that we get an empty array when request is valid, but review has no comments yet